README.md

```diff
@@ -0,0 +1,28 @@
+# TiKV RFCs
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the TiKV community.
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the project, so that all stakeholders can be confident about the direction the project is evolving in.
+
+### How to submit an RFC
+
```

Comment:
Do we need to assign a reviewer to owner to each RFC?

Reply:
em, I have been considering this, mostly, I think if two of our team guys agree, the RFC is approved.

But maybe it is better to form a committee and the RPC can be approved if and only if all the guys agree on it.

Reply:
you can see the timeline below, I prefer two maintainers approving. But now, the maintainer is only in PingCAP team, so we need to attract outside contributors, promote some to the maintainer.

Reply:
Yes, exactly!